src/gp_creds.c

```diff
@@ -492,6 +492,7 @@ static int gp_get_cred_environment(struct gp_call_ctx *gpcall,
 }
 
 static uint32_t gp_check_cred(uint32_t *min,
+                              struct gp_call_ctx *gpcall,
```

Comment:
You are not using gpcall, please pass in a pointer to gp_service directly instead.

Reply:
Okay.